docs/production/export-and-import.md

```diff
@@ -106,6 +106,17 @@ Until you do, your Zulip server will think its user-facing hostname is
 still `zulip.example.com` and will return HTTP `400 BAD REQUEST`
 errors when trying to access it via `zuliptest.example.com`.
 
+#### Changing database settings
+
+If you wish to restore onto a very differently configured host (e.g. with `REMOTE_POSTGRES_HOST` set to a different value), you can edit `/etc/zulip/settings.py` to configure the host to suit the new host's needs, then restore with `--keep-settings`:
```

Comment:
This should be line-wrapped.

Reply:
Done!  Dunno how I didn't do that originally.